CONTRIBUTING.md

```diff
@@ -80,7 +87,7 @@ You can [develop locally](#developing-locally) or use an online coding developme
 
 1. **Create a new codespace** via https://codespaces.new/withastro/starlight
 
-2. If running the docs site, pass the `--host` flag to avoid “502 Bad Gateway” errors:
+2. **Run the dev server** to generate types. Pass the `--host` flag to avoid “502 Bad Gateway” errors:
```

Comment:
If we add an extra step to this section dedicated to types before this step, we should be able to revert these changes and keep this step focused on the `--host` requirement when and if running the docs site.

```suggestion
2. If running the docs site, pass the `--host` flag to avoid “502 Bad Gateway” errors:
```